dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -27,7 +27,7 @@
 .script_text
   = text_area_tag('level[dsl_text]', @level.dsl_text || (@level.dsl_default))
 
-- if [ContractMatch, Match, Multi, TextMatch, External].include? @level.class
+- if [ContractMatch, Match, Multi, TextMatch, External, LevelGroup].include? @level.class
```

Comment:
@bcjordan I think markdown support was from you?  I notice that if I don't include LevelGroup in this list and therefore ensure that .markdown is included on the page, then I get error "code-studio-common.js:14 Uncaught TypeError: Cannot read property 'value' of null" in levelbuilder_dsl.js at levels/new?type=LevelGroup.  Is this by design, or should we move the include of levelbuilder_dsl.js into the body of this condition?